s3/pkg/meta/db/drivers/tidb/bucket.go

```diff
@@ -112,6 +112,14 @@ func (t *TidbClient) GetBucket(ctx context.Context, bucketName string) (bucket *
 		tmp.Versioning.Status = versionOpts.Status
 	}
 
+	// get SSE info for this bucket
+	sseOpts, sseErr := t.GetBucketSSE(ctx, tmp.Name)
```

Comment:
it seems that a separate bucket sse table is used for bucket sse info?

Reply:
correct